src/frontend/komponenter/Fagsak/Mottaker/RegistrerMottaker.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+
+import styled from 'styled-components';
+
+import { Alert } from '@navikt/ds-react';
+import { RessursStatus } from '@navikt/familie-typer';
+
+import { useBehandling } from '../../../context/behandlingContext/BehandlingContext';
+import { useMottakerType } from '../../../context/MottakerTypeContext';
+import { BehandlingSteg } from '../../../typer/behandling';
+import { FagsakType } from '../../../typer/fagsak';
+import Skjemasteg from '../../Felleskomponenter/Skjemasteg/Skjemasteg';
+import Institusjon from './Institusjon';
+import Verge from './Verge';
+
+const StyledSkjemasteg = styled(Skjemasteg)`
+    max-width: 40rem;
+`;
+
+const StyledAlert = styled(Alert)`
+    margin: 2rem;
+`;
+
+const RegistrerMottaker: React.FC = () => {
```

Comment:
We are using InstitusjonOgVerge in the backend, but Mottaker in the frontend, this might be a bit confusing for people who is new to the code. Is it possible that we use the same term?

Reply:
Yes, I have renamed the term Mottaker to InstitusjonOgVerge for the most part. Wasn't sure what to do about the name above, though, since it just reflects the title displayed for the component in frontend